README.rst

```diff
@@ -47,15 +47,18 @@ The built html pages can be found in ``doc/_build/html/``
 Releasing
 ---------
 
-Manually for now... see the todo below for how we hope to eventually do it 
+Manually for now... see the todo below for how we hope to eventually do it
 automagically.
 
+- be sure to edit `mpl_sphinx_theme/_version.py`
+
 .. code-block::
 
    $ git checkout <commit-hash>
    $ git tag -a x.y.z -m 'Version x.y.z'
    $ git push upstream main --tags
-   $ python -m build -s -w
+   $ python setup.py sdist
+   $ python setup.py bdist_wheel
```

Comment:
Did `build` not create both of them correctly? That is the new way going forward (to hide or even drop `setup.py` eventually.)

Reply:
I guess I don't have build? What is it?

Reply:
OK, `mamba install build` works fine ;-)

Reply:
It's a "PEP 517 build frontend", that is, it reads the build system defined in `pyproject.toml`, creates an isolated environment for it, and builds the sdist in there. So essentially it should work everywhere regardless of whether it's `setup.py`, or flit, or hatch, etc. Of course, it doesn't make too much difference for `setup.py`-based projects, but it should be more portable.